rsconnect/actions_content.py

```diff
@@ -81,29 +87,29 @@ def build_remove_content(
     :return: A list of guids of the content items that were removed
     """
     init_content_build_store(connect_server)
-    if _content_build_store.get_build_running():
+    if content_build_store().get_build_running():
         raise RSConnectException(
             "There is a build running on this server, " + "please wait for it to finish before removing content."
         )
     guids: list[str] = [guid]
     if all:
-        guids = [c["guid"] for c in _content_build_store.get_content_items()]
+        guids = [c["guid"] for c in content_build_store().get_content_items()]
     for guid in guids:
-        _content_build_store.remove_content_item(guid, purge)
+        content_build_store().remove_content_item(guid, purge)
```

Comment:
What do you think about changing `init_content_build_store()` to a signature like this, where inits if necessary, and then always returns the store:

```py
def ensure_content_build_store(connect_server: RSConnectServer) -> ContentBuildStore:
    global _content_build_store
    if not _content_build_store:
        logger.info("Initializing ContentBuildStore for %s" % connect_server.url)
        _content_build_store = ContentBuildStore(connect_server)
    return _content_build_store
```

Then in usage in the various functions, it would look like this:

```py
build_store = ensure_content_build_store()
```